.github/workflows/CI.yml

```diff
@@ -17,8 +17,9 @@ jobs:
     strategy:
       matrix:
         include:
-          - {os: ubuntu-18.04, cc: gcc-7, cxx: g++-7}
-          - {os: ubuntu-18.04, cc: clang-8, cxx: clang++-8}
+          - {os: ubuntu-20.04, cc: gcc-10, cxx: g++-10}
+          - {os: ubuntu-20.04, cc: clang-12, cxx: clang++-12}
```

Comment:
Triqs 3.1 requires clang 13+.
See e.g. https://github.com/TRIQS/app4triqs/blob/3.1.x/.github/workflows/build.yml for the app4triqs CI action.